docs/source/jsonpath.rst

```diff
@@ -7,10 +7,14 @@ validating response bodies that are formatted as JSON and making
 reference to that JSON in subsequent queries. `jsonpath_rw`_ is used
 to process the JSONPath expressions.
 
-To address a common requirement when evaluting JSON responses, an
-extension has been made to the default implementation of JSONPath.
-This extension is ``len`` and will return the length of the current
-datum in the JSONPath expression.
+To address common requirements when evaluting JSON responses,
+extensions have been made to the default implementation of JSONPath
+using `jsonpath_rw_ext`. It adds three useful functions:
+
+#. Return the length of the current datum using ``len``.
+#. Sort the current datum using ``sorted`` and ``[/name]``.
```

Comment:
Yeah, I would like to make the docs for all of the json path features much more complete but I didn't want needing that to hold up the release (I needed the feature working before this coming wednesday). It's probably time for another robust run through the docs. I may have time for that now that I'm past the crunch time for nova (which resulted in utter failure in anything getting merged).